src/cls/2pc_queue/cls_2pc_queue.cc

```diff
@@ -578,6 +578,19 @@ static int cls_2pc_queue_list_entries(cls_method_context_t hctx, bufferlist *in,
   return 0;
 }
 
+static uint64_t cls_2pc_queue_count_entries(cls_method_context_t hctx, cls_queue_list_op& op, cls_queue_head& head)
+{
+  cls_queue_list_ret op_ret;
+  auto ret = queue_list_entries(hctx, op, op_ret, head);
+  if (ret < 0) {
+    return ret;
```

Comment:
you cannot do that for unsigned int. best option is to have the return value be "int", indicating error with negatives and 0 as ok, and return the length by value